python/plugins/processing/algs/qgis/RasterCalculator.py

```diff
@@ -88,6 +89,9 @@ def clone(self):
         self.addParameter(QgsProcessingParameterCrs(self.CRS, 'Output CRS', optional=True))
         self.addParameter(QgsProcessingParameterRasterDestination(self.OUTPUT, self.tr('Output')))
 
+    def flags(self):
+        return super().flags() | QgsProcessingAlgorithm.FlagDeprecated | QgsProcessingAlgorithm.FlagNotAvailableInStandaloneTool
```

Comment:
We better leave this available for use, in case some existing scripts utilise it:

```suggestion
        return super().flags() | QgsProcessingAlgorithm.FlagDeprecated
```